server/home/views.py

```diff
@@ -52,3 +53,50 @@ def search(request):
             })
 
     return render(request, 'home/search.html', {'query': query_text, 'results': results})
+
+
+# =================================================================================================
+
+
+def home(request):
+    return render(request, 'home/home.html')
+
+
+def organization(request):
+    details = MetaData.objects.all()
+    return render(request, 'home/organization.html',{'details':details})
+
+
+def searchresult(request):
+    dropdown_values = []
+    dropdown_values_unique = ""
+
+    field = request.GET.get('field')
+    data = request.GET.get('data')
+    print(field, "fieldddddddd")
+    print(data,"ddddddddddddddddata")
+
+    if field:
+        dropdown_values = MetaData.objects.filter(**{f"{field}__isnull": False}).values_list(field, flat=True)
+        dropdown_values_lower = []
+
+        for value in dropdown_values:
+            if isinstance(value, list):
+                dropdown_values_lower.extend([item.lower() for item in value])
+            else:
+                dropdown_values_lower.append(value.lower())
+
+        dropdown_values_unique = set(dropdown_values_lower)
+        print(dropdown_values_unique, "unique valueeeeeeeeees")
+
+        return JsonResponse({"dropdown_values": list(dropdown_values_unique)})
+
+    field_names = ["language", "states"]    
+    messages = ['language:malayalam', 'state:kerala', 'organization: pradan']
```

Comment:
Should not be hardcoded. Options can be read from https://github.com/rohittp0/vanswer/blob/7f02038e909d0d89116f85389c77de508aaac21d/server/home/constants.py#L1